Solutions/AliCloud/DataConnectors/azuredeploy_Connector_AliCloud_API_AzureFunction.json

```diff
@@ -0,0 +1,209 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "AliCloud",
+            "minLength": 1,
+            "maxLength": 11,
+            "type": "string"
+        },
+        "WorkspaceID": {
+            "type": "string",
+            "defaultValue": "<workspaceID>"
+        },
+        "WorkspaceKey": {
+            "type": "securestring",
+            "defaultValue": "<workspaceKey>"
+        },
+        "AliCloudAccessKeyId": {
+            "type": "string",
+            "defaultValue": "<AliCloudAccessKeyId>"
+        },
+        "AliCloudAccessKey": {
+            "type": "securestring",
+            "defaultValue": "<AliCloudAccessKey>"
+        },
+        "AliCloudProjects": {
+            "type": "description",
+            "defaultValue": ""
```

Comment:
Please change the type "description" to "string", for parameters AliCloudProjects and AliCloudworkers. As it is throwing the deployment error.

![image](https://user-images.githubusercontent.com/105280229/178964194-2fc7d062-b178-4a9a-96b5-4a2fc17d4a55.png)

Reply:
@VenkataPrathibha, done.  Changes are made in the PR - https://github.com/Azure/Azure-Sentinel/pull/5629.